src/CMakeLists.txt

```diff
@@ -25,6 +26,7 @@ set(SRC
     stdlib_experimental_error.f90
     stdlib_experimental_kinds.f90
     stdlib_experimental_optval.f90
```

Comment:
```suggestion
```
Source files generated by `fypp` are part of the variable `${outFiles}` and need not be listed here. (same goes for `io` and `stats`)